Update add-staff.md (#10309)
MicrosoftDocs/microsoft-365-docs: microsoft-365/bookings/add-staff.md

    ```powershell
    Add-MailboxPermission -Identity <bookingmailbox@emailaddress> -User <adminusers@emailaddress> -AccessRights FullAccess -Deny:$false
    ```
    

>     By default, the mailbox gets automatically mapped to outlook. If you donot want the booking mailbox to show in users outlook, assign full access with the following commands:

Reviewer: ```suggestion
    By default, the mailbox gets automatically mapped to Outlook. If you do not want the booking mailbox to appear in the user's Outlook, assign full access with the following commands:
```

    ```powershell
    Add-MailboxPermission -Identity "daycare@contoso.com" -User "Allie Bellew" -AccessRights FullAccess -InheritanceType All
    ```

>      By default, the mailbox gets automatically mapped to outlook. If you donot want the booking mailbox to show in users outlook, assign full access with the following commands:

Reviewer: ```suggestion
     By default, the mailbox gets automatically mapped to Outlook. If you do not want the booking mailbox to appear in the user's Outlook, assign full access with the following commands:
```

>      By default, the mailbox gets automatically mapped to Outlook. If you do not want the booking mailbox to appear in the user's Outlook, assign full access with the following commands:

Reviewer: ```suggestion
     By default, the mailbox gets automatically mapped to Outlook. If you do not want the booking mailbox to appear in the user's Outlook, assign full access with the following commands:
```

>      By default, the mailbox gets automatically mapped to Outlook. If you do not want the booking mailbox to appear in the user's Outlook, assign full access with the following commands:

Author: @JohanFreelancer9 I don't see the option to Commit
